lightning/src/ln/functional_tests.rs

```diff
@@ -3668,6 +3673,17 @@ fn do_test_drop_messages_peer_disconnect(messages_delivered: u8) {
 	nodes[0].node.peer_disconnected(&nodes[1].node.get_our_node_id(), false);
 	nodes[1].node.peer_disconnected(&nodes[0].node.get_our_node_id(), false);
 	if messages_delivered < 3 {
+		if simulate_broken_lnd {
+			// lnd has a long-standing bug where they send a funding_locked prior to a
+			// channel_reestablish if you reconnect prior to fucking_locked time.
```

Comment:
sorry for the nit, but is that a typo lol?

Reply:
Ha, indeed it is, and definitely not a nit, that one lol

Reply:
That's hilarious lol